codec-http/src/main/java/io/netty/handler/codec/http/CookieUtil.java

```diff
@@ -90,7 +96,74 @@ static void add(StringBuilder sb, String name, long val) {
         sb.append((char) HttpConstants.SP);
     }
 
-    private CookieEncoderUtil() {
+    private static final BitSet VALID_COOKIE_VALUE_OCTETS = validCookieValueOctets();
```

Comment:
BitSet is since JDK1.0 ;)
https://docs.oracle.com/javase/7/docs/api/java/util/BitSet.html